ui/pages/swaps/build-quote/build-quote.js

```diff
@@ -627,44 +627,41 @@ export default function BuildQuote({
                   alt={t('swapSwapSwitch')}
                 />
               </Box>
-              <Typography variant={TypographyVariant.H7} marginTop={0}>
+              <Text variant={TextVariant.bodySm} as="h6" marginTop={0}>
                 {t('stxDescription')}
-              </Typography>
-              <Typography
-                as="ul"
-                variant={TypographyVariant.H7}
-                fontWeight={FONT_WEIGHT.BOLD}
-                marginTop={3}
-              >
-                <li>{t('stxBenefit1')}</li>
-                <li>{t('stxBenefit2')}</li>
-                <li>{t('stxBenefit3')}</li>
-                <li>
-                  {t('stxBenefit4')}
-                  <Typography
-                    as="span"
-                    fontWeight={FONT_WEIGHT.NORMAL}
-                    variant={TypographyVariant.H7}
-                  >
-                    {' *'}
-                  </Typography>
-                </li>
-              </Typography>
-              <Typography
-                variant={TypographyVariant.H8}
+              </Text>
+              <Text as="ul" variant={TextVariant.bodySm} marginTop={3}>
+                <strong>
+                  <li>{t('stxBenefit1')}</li>
+                  <li>{t('stxBenefit2')}</li>
+                  <li>{t('stxBenefit3')}</li>
+                  <li>
+                    {t('stxBenefit4')}
+                    <Text
+                      as="span"
+                      fontWeight={FontWeight.Normal}
+                      variant={TextVariant.bodySm}
+                    >
+                      {' *'}
+                    </Text>
+                  </li>
+                </strong>
+              </Text>
```

Comment:
```suggestion
              <Text as="ul" variant={TextVariant.bodySmBold} marginTop={3}>
                  <li>{t('stxBenefit1')}</li>
                  <li>{t('stxBenefit2')}</li>
                  <li>{t('stxBenefit3')}</li>
                  <li>
                    {t('stxBenefit4')}
                    <Text
                      as="span"
                      fontWeight={FontWeight.Normal}
                      variant={TextVariant.bodySm}
                    >
                      {' *'}
                    </Text>
                  </li>
              </Text>
```

Reply:
Since we apply bold to all the elements inside this component, we can directly add style in <Text> component

Reply:
Yes @DDDDDanica ,you are right.

I corrected this in another PR when @brad-decker mentioned, forgot to do that here.

Thanks you.